src/test/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/uploaddocuments/UploadDocumentFurtherEvidenceHandlerTest.java

```diff
@@ -74,7 +76,20 @@ public void handleHappyPathWhenAllFieldsAreProvided() throws IOException {
         PreSubmitCallbackResponse<SscsCaseData> actualCaseData = handler.handle(ABOUT_TO_SUBMIT, callback,
             USER_AUTHORISATION);
 
-        assertThatJson(actualCaseData).isEqualTo(getExpectedResponse());
+        assertThatJson(actualCaseData).isEqualTo(getExpectedResponse(UPLOAD_DOCUMENT_FE_EXPECTED_JSON));
+        assertEquals("feReceived", actualCaseData.getData().getDwpState());
+        assertNull(actualCaseData.getData().getDraftSscsFurtherEvidenceDocument());
+    }
+
+    @Test
+    public void handleHappyPathWhenAudioVideoFileUploaded() throws IOException {
+        Callback<SscsCaseData> callback = buildTestCallbackGivenData(UPLOAD_DOCUMENT_FURTHER_EVIDENCE,"appealCreated",
+                "Other evidence", "appellantEvidence", UPLOAD_AUDIO_VIDEO_DOCUMENT_FE_CALLBACK_JSON);
+
+        PreSubmitCallbackResponse<SscsCaseData> actualCaseData = handler.handle(ABOUT_TO_SUBMIT, callback,
+                USER_AUTHORISATION);
+
+        assertThatJson(actualCaseData).isEqualTo(getExpectedResponse(UPLOAD_AUDIO_VIDEO_DOCUMENT_FE_EXPECTED_JSON));
```

Comment:
It might be  much better asserting there are no documents in sscsDocuments and there is a document in the audio visual collection, rather than asserting the whole JSON document.